sql/timeseries/cruxPassesCWV.sql

```diff
@@ -15,8 +15,8 @@ SELECT
   IF(device = 'desktop', 'desktop', 'mobile') AS client,
   SAFE_DIVIDE(
     COUNT(DISTINCT IF(
-      /* FID can be null and is not mandatory for CWV */
-      (p75_fid IS NULL OR IS_GOOD(fast_fid, avg_fid, slow_fid)) AND
+      /* INP can be null and is not mandatory for CWV */
+      (p75_inp IS NULL OR IS_GOOD(fast_inp, avg_inp, slow_inp)) AND
```

Comment:
Should we have a `CASE` statement to look at the appropriate field before and after March 2024?

We normally run this report in incremental mode, but if we do a full run, then it will overwrite all the old data.

Reply:
Updated. I wonder if that might actually confuse the script that turns full runs into incremental runs.

Reply:
Actually if I'm reading this correctly, we will always run the full CrUX queries instead of incremental (so your suggestion is necessary):

https://github.com/HTTPArchive/bigquery/blob/master/sql/generate_reports.sh#L279

Still looking to see if we need to worry about the transforms for the lenses.

Reply:
> Actually if I'm reading this correctly, we will always run the full CrUX queries instead of incremental (so your suggestion is necessary):
> 
> https://github.com/HTTPArchive/bigquery/blob/master/sql/generate_reports.sh#L279

Godo spot!

> Doesn't look like we need to worry about the lens transforms:
> 
> https://github.com/HTTPArchive/bigquery/blob/master/sql/generate_reports.sh#L330-L348
> 
> AFAICT it only checks for dates _after_ the `WHERE` clause.

Yeah thing it should be fine not to edit there.